### ISSUE TITLE
PrintJobOperations | newPrintJob | Update employee lookup to only use employees already in the database

### ISSUE THREAD

**Issue by jberns** [codefordenver/ideaLab#165]
Acceptance Criteria:
- [ ] Remove creating a new database email if it does not exist (This was originally for testing only) and change to throwing error like the `findPrintJobById` example below.
```    
        if (databaseEmail == null) {
            databaseEmail = new EmailHash(emailHash); <Remove
            databaseEmail = emailHashRepo.save(databaseEmail); <Remove
        }
```
- [ ] If `databaseEmail == null` throw a new error named `EMAIL_CANT_FIND_BY_HASH`
- [ ] Add the new error the to the exception/ErrorType file as the next enumeration number with a `NOT FOUND` status
- [ ] Add test for employee already existing in database - Test should return a print job without an error.
- [ ] Add test for employee not existing in the database - Test should return the `EMAIL_CANT_FIND_BY_HASH` error.

Example of throwing an error:
```
       PrintJob printJob = printJobRepo.findPrintJobById(printId);
       if(printJob == null) {
            throw new IdeaLabApiException(PRINT_JOB_CANT_FIND_BY_ID);
        }
```

**Comment by rmatsunaga** [comment]
Hi is this issue still open? I'd love to contribute!

**Comment by jberns** [comment]
@rmatsunaga yes, I will write some more detailed acceptance criteria over the weekend and you can work on this next week.

**Comment by rmatsunaga** [comment]
thank you so much! looking forward to it

**Comment by jberns** [comment]
Hi @rmatsunaga I added more details and the hacktoberfest label if you were doing it 👍

**Comment by rmatsunaga** [comment]
Hi, I'm trying to setup a testing environment and haven't finished setting the platform itself up. I can't seem to find the sample_dropbox.properties file I need to finish setting up the environment

**Comment by rmatsunaga** [comment]
Hi, I'm having trouble running the tests, does anyone know why the test run arrows aren't showing up in my intelliJ IDE?

**Comment by jberns** [comment]
@rmatsunaga My arrows show up, you can try right clicking the tests folder and then run them all from the explorer actions.  I'm not sure how else to help on that issue.

<img width="507" alt="Screen Shot 2019-10-10 at 4 48 50 PM" src="https://user-images.githubusercontent.com/519070/66612131-0394c980-eb7e-11e9-9c5f-976d988e1b41.png">

**Comment by jberns** [comment]
Closed through #214
